client/app/components/BeaconConsent.jsx

```diff
@@ -76,8 +75,4 @@ export function BeaconConsent() {
   );
 }
 
-export default function init(ngModule) {
-  ngModule.component('beaconConsent', react2angular(BeaconConsent));
-}
-
-init.init = true;
+export default BeaconConsent;
```

Comment:
Just wondering: why do you prefer this pattern (declare component and export it at the end of file) instead of immediately exporting (`export default function BeaconConsent(...)`)?

Reply:
I don't have a strong reasoning for that, both are fine to me. I usually go with this pattern because at some point I got used to see those exports in the end of files. I guess you prefer them immediately exported :laughing:?

Reply:
> I guess you prefer them immediately exported

Yes :smile: (but it's just because I used to use this variant, I think both are fine)